conf/study-defaults/email-sign-in.txt

```diff
@@ -1 +1 @@
-Click here to sign in: <a href="https://${host}/mobile/startSession.html?email=${email}&study=${studyId}&token=${token}">https://${host}/mobile/startSession.html?email=${email}&study=${studyId}&token=${token}</a>
+Click here to sign in: ${url}
```

Comment:
Do we need the a href tag?

Reply:
No, none of the others have it and most email clients automatically link URLs when they are presented. However, it looks like users can add them in the editor and they work, if they want to do it (this was not true with Stormpath IIRC).